tests/Feature/OrderCancellation/OrganisationWithoutTaxTest.php

```diff
@@ -164,19 +147,12 @@ public function cancels_and_refunds_order_with_single_ticket_with_percentage_boo
     public function cancels_and_refunds_order_with_multiple_tickets_with_percentage_booking_fees()
     {
         // Setup single attendee order with % fees
-        list($order, $attendees) = $this->makeTicketOrder(5, 100.00, true);
+        [$order, $attendees] = $this->makeTicketOrder(5, 100.00, true);
         $attendeeIds = $attendees->pluck('id')->toArray();
         $response = $this->actingAs($this->getAccountUser())
             ->post("event/order/$order->id/cancel", [
-                'attendees' => [ $attendeeIds[0] ],
+                'attendees' => [$attendeeIds[0]],
```

Comment:
Sorry to be pedantic again. This change is breaking the test logic. In this test we are testing multiple attendees getting cancelled so please swap out the single attendee for the multiple like below:

```
                'attendees' => [
                    $attendeeIds[0],
                    $attendeeIds[1],
                    $attendeeIds[2],
                ],
```